src/main/java/gameframework/drawing/Drawable.java

```diff
@@ -2,6 +2,10 @@
 
 import java.awt.Graphics;
 
+/**
+ * Classes implementing this interface can be drawn on the Canvas, using the method draw
```

Comment:
You mention a Canvas, but I don't see that in the code.
Maybe explain when those objects will be drawn?